lib/qesapdeployment.pm

```diff
@@ -743,7 +744,7 @@ sub qesap_get_az_resource_group {
     my (%args) = @_;
     my $substring = $args{substring} ? " | grep $args{substring}" : "";
     my $job_id = get_current_job_id();
-    my $result = script_output("az group list --query \"[].name\" -o tsv | grep $job_id" . $substring);
+    my $result = script_output("az group list --query \"[].name\" -o tsv | grep $job_id" . $substring, proceed_on_failure => 1);
```

Comment:
Note for a next PR: add a unit test in `t/09_qesapdeployment.t` in case of `GYROS` is finished ;-)